### PR TITLE
first PR for number_of_days()

### PR DESCRIPTION
**Pull request by viettran15**
### Title
Pull Request for number_of_days() function. 

### Summary
number_of_days()
This function returns the total days in a specific month of the year.

### Details
Input: (year, month)
Output: number of days within the specified month of a given year

Example1:
Input: (2023, 1)
Output: 31

Example2:
Input: (2024, 2)
Output: 29

### CODE REVIEW

File: Calendar_CLI/Calendar.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    """
+    This function returns the total days in a specific month of the year.
+    
+    args:
+        year: integer
+        month: integer between 1 and 12
+    """
+
+    # Returns the number of days in the month depending on the year and month. 
+    if (month == 2) and ((year%4 == 0) or ((year%100 == 0) and (year%400 == 0))):
```
Comment (alex308248) [inline Calendar_CLI/Calendar.py]: The if case here have an issue. You are going to return 29 for any year%4 == 0.

File: Calendar_CLI/Calendar.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+def number_of_days(year, month):
+    """
+    This function returns the total days in a specific month of the year.
+    
+    args:
+        year: integer
```
Comment (alex308248) [inline Calendar_CLI/Calendar.py]: What if the year is negative? Don't forget to handle the cases at BC.

**Review by alex308248 — CHANGES_REQUESTED** [review]
Need to handle the cases in BC. The if-else cases is also wrong

File: Calendar_CLI/Calendar.py (hunk 1 above)
Comment (viettran15) [reply Calendar_CLI/Calendar.py]: I've updated this case to be return 29 if (year%4 == 0) AND (year%100 != 100)

File: Calendar_CLI/Calendar.py (hunk 2 above)
Comment (viettran15) [reply Calendar_CLI/Calendar.py]: I've added cases to handle years BC as well as if the user enters 0 (invalid year).

**Review by alex308248 — APPROVED** [review]
Well done!

**Comment by alex308248** [comment]
Don't forget to fix the conflicts and merge the branch

**Pull request merged**